<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentCreator</class>
 <widget class="QDialog" name="studentCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>634</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>634</width>
    <height>404</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生创建巫师</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>511</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="basic_info">
    <attribute name="title">
     <string>基本信息</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>学号*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="id_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>姓名*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>密码*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pwd_edit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>再次输入密码*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="pwd_makesure_edit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="contacts">
    <attribute name="title">
     <string>联系方式</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>出生日期</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="adress"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="gender_combo">
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>跨性别</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>电话</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="telephone_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>电子邮件</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="email_edit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请确认信息填写正确</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提示:带*号的信息必须填写</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>158</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="quit_button">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
